web/hooks/use-has-created-contract-today.ts

```diff
@@ -4,16 +4,20 @@ import { User } from 'common/user'
 
 let sessionCreatedContractToday = true
 
+export function getUtcMidnightToday() {
+  // Uses utc time like the server.
+  const utcTimeString = new Date().toISOString()
+  return new Date(utcTimeString).setUTCHours(0, 0, 0, 0)
```

Comment:
Actually, if we're going to pick a hardcoded time, I might try 4PM UTC! It's noon in ET, and 9am in PT; I think US/UK is our main audience for now, and it might be cool & habit-forming to have the countdown when everyone is awake/online; could imagine everyone logging on at the same time each day and betting on all the new markets haha

Reply:
(On balance, I'm leaning (70%) "user's local midnight" to still be the best design, but it's a lot more work afaict)